### PR TITLE
Feature Card

### PR DESCRIPTION
**Pull request by Avin008**
### Card Component Includes:

- Text Only Card

basic card

- Horizontal Cards with and without Badges

horizontal card

- Vertical Cards with and without Badges and Close Button

vertical card

- Text Overlay Card

overlay-card

### CODE REVIEW

File: Components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+.overlay-text {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgb(234, 234, 236, 0.6);
```
Comment (sushil-mod) [inline Components/Card/card.css:144]: if possible use color variable !

**Review by sushil-mod — APPROVED** [review]

File: Components/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,187 @@
+/* card common styles */
+
+.card {
+  display: flex;
+  gap: 2rem;
+  box-shadow: 1px 1px 2px gray;
+  border-radius: 0.5rem;
+  padding: 1rem;
+  font-family: "Montserrat", sans-serif;
+  background-color: whitesmoke;
```
Comment (ozairkhan2810) [inline Components/Card/card.css:10]: use color variable

File: Components/Card/card.css [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  position: relative;
+}
+
+.card__img {
+  height: 100%;
+  width: 100%;
+}
+
+.card__body {
+  max-width: 400px;
```
Comment (ozairkhan2810) [inline Components/Card/card.css:25]: use rem for consistency

File: Components/Card/card.css [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+}
+
+.card__description {
+  font-weight: 500;
+}
+
+.card__price--stricked {
+  color: #ff527b;
```
Comment (ozairkhan2810) [inline Components/Card/card.css:33]: use color variable

File: Components/Card/card.css [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+  gap: 1rem;
+}
+
+.card__btn {
+  box-sizing: border-box;
+  padding: 0.5rem 0.8rem;
+  border-radius: 0.5rem;
+  text-decoration: none;
+  color: white;
+  background-color: #ff527b;
```
Comment (ozairkhan2810) [inline Components/Card/card.css:50]: use color variable

**Pull request merged**